app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -141,6 +141,7 @@ def parse_vm(vm)
       :name                => name,
       :vendor              => "vmware",
       :raw_power_state     => status,
+      :snapshots           => parse_snapshot(vm),
```

Comment:
```ruby
:snapshots => [parse_snapshot(vm)].compact,
```

Nitpicking, but can you please use such format ^, it's annoying if we say "parse (single) snapshot" and then return a list :)